content/packages/working-with-a-github-packages-registry/working-with-the-rubygems-registry.md

```diff
@@ -52,7 +46,12 @@ shortTitle: RubyGems registry
 
 {% data reusables.package_registry.required-scopes %}
 
-You can authenticate to {% data variables.product.prodname_registry %} with RubyGems by editing the  *~/.gem/credentials* file for publishing gems, editing the *~/.gemrc* file for installing a single gem, or using Bundler for tracking and installing one or more gems.
+You can authenticate to {% data variables.product.prodname_registry %} with RubyGems by: 
+* editing the  *~/.gem/credentials* file for publishing gems
+* adding a source using `gem sources --add` for installing gems using `gem install`
+* using Bundler for installing gems from an application Gemfile
+
+#### Publishing Gems
```

Comment:
```suggestion
To publish and install gems, you can configure RubyGems or Bundler to authenticate to {% data variables.product.prodname_registry %} using your personal access token.
```

For further clarity, let's make this text more of an introductory summary, and focus on the details later in the section.